fastai/tabular/data.py

```diff
@@ -12,22 +12,39 @@ class TabularDataLoaders(DataLoaders):
     "Basic wrapper around several `DataLoader`s with factory methods for tabular data"
     @classmethod
     @delegates(Tabular.dataloaders, but=["dl_type", "dl_kwargs"])
-    def from_df(cls, df, path='.', procs=None, cat_names=None, cont_names=None, y_names=None, y_block=None,
-                valid_idx=None, **kwargs):
-        "Create from `df` in `path` using `procs`"
+    def from_df(cls,
+                df:pd.DataFrame, # A Dataframe object
```

Comment:
```suggestion
                df:pd.DataFrame, 
```
No need to point out it is a dataframe object since it is named df and has type pd.DataFrame